tutorials/models/2_small_graph/3_tree-lstm.py

```diff
@@ -88,7 +88,7 @@ def plot_tree(g):
             node_color=[[.5, .5, .5]], arrowsize=4)
     plt.show()
 
-plot_tree(graph.to_networkx())
+#plot_tree(graph.to_networkx())

```

Comment:
Why comment out?

Reply:
I forget to uncomment it. Used in local test